test/js/node/stream/node-stream.test.js

```diff
@@ -386,6 +386,27 @@ it("Readable.fromWeb", async () => {
   expect(Buffer.concat(chunks).toString()).toBe("Hello World!\n");
 });
 
+it("Writable.toWeb", async () => {
```

Comment:
Where are the tests for Writable.fromWeb?

Reply:
added!